### DISCUSSION TITLE
Using GCS instead of minio with milvus on GKE

### DISCUSSION THREAD

**Discussion by yashgovilkar** [milvus-io/milvus#26305] [Q&A and General discussion]
Hi, I have deployed a milvus cluster on GKE using the instructions provided in [this](https://milvus.io/docs/gcp.md) doc.  
I want to use GCS as my persistent daata storage bucket instead  of minio. 
I tried enabling the GCS gateway in values.yaml, but I got 
``` 
ERROR Gateway is deprecated, To continue to use Gateway please use releases no later than 'RELEASE.2022-10-24T18-35-07Z'. We recommend all our users to migrate from gateway mode to server mode. Please read https://blog.min.io/deprecation-of-the-minio-gateway/: Invalid arguments specified
``` 
Any help on how to replace minio with gcs bucket would be much appreciated.
@yhmo

**Comment by xiaofan-luan** [comment]
@zwd1208 could you help on it
